src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -721,6 +722,24 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 _cursorTimer = std::nullopt;
             }
 
+            // Set up blinking attributes
+            BOOL animationsEnabled = true;
+            SystemParametersInfoW(SPI_GETCLIENTAREAANIMATION, 0, &animationsEnabled, 0);
+            if (animationsEnabled && blinkTime != INFINITE)
+            {
+                // Create a timer
+                DispatcherTimer blinkTimer;
+                blinkTimer.Interval(std::chrono::milliseconds(blinkTime));

```

Comment:
Can the `blinkTimer` and `cursorTimer` be merged into one unified `animationTimer`? I mean they all tick at `blinkTime` interval. They share most of their constructor & deconstruction logic. And Maybe `cursorTimer` should also consider `SPI_GETCLIENTAREAANIMATION`?

Reply:
No, I mentioned this in the PR notes. The cursor timer turns off whenever keys are pressed, so if you're typing something on the command line, then suddenly all the blinking attributes on the screen will freeze. I did try using it, but it definitely didn't feel right.

Reply:
I appreciate that they are separate timers. Thank you for that.